modules/sdk-core/src/bitgo/wallet/wallet.ts

```diff
@@ -1986,38 +1986,85 @@ export class Wallet implements IWallet {
    * @param params
    */
   async accelerateTransaction(params: AccelerateTransactionOptions = {}): Promise<any> {
-    // TODO(BG-9349): change the last check to > 0 and the error message once platform allows multiple transactions to
-    //                be bumped in the same CPFP transaction
-    if (_.isUndefined(params.cpfpTxIds) || !Array.isArray(params.cpfpTxIds) || params.cpfpTxIds.length !== 1) {
-      const error: any = new Error('expecting cpfpTxIds to be an array of length 1');
-      error.code = 'cpfptxids_not_array';
+    this.validateAccelerationParams(params);
+
+    params.recipients = [];
+
+    return await this.submitTransaction({
+      ...(await this.prebuildAndSignTransaction(params)),
+      ...BuildParams.encode(params),
+    });
+  }
+
+  private validateAccelerationParams(params: AccelerateTransactionOptions) {
+    if (!params.cpfpTxIds && !params.rbfTxIds) {
+      const error: any = new Error('must pass cpfpTxIds or rbfTxIds');
+      error.code = 'cpfptxids_or_rbftxids_required';
```

Comment:
Is this user facing error? Will they see this exact string?

Reply:
They will see the error message. Looks like we use the code for internal purposes mostly.